contracts/libraries/SafeToL2Setup.sol

```diff
@@ -16,7 +16,7 @@ contract SafeToL2Setup is SafeStorage {
      * @notice Address of the contract.
      * @dev This is used to ensure that the contract is only ever `DELEGATECALL`-ed.
      */
-    address public immutable _SELF;
+    address public immutable SELF;
```

Comment:
Instead, can we change this to be `private`?

Reply:
Also, this wasn't included in the findings right? Why are we changing it then? (although I do still think it should be `private`)

Reply:
This was an open question: https://github.com/safe-global/safe-smart-account/pull/812#issuecomment-2304291497

As we were suggested (not applied) some style-based changes, thought this was a small one to include in this. Could be done in the PR with the `main` though.

Reply:
I think the style change is good for consistency, but we should make the varible private anyhow.

Reply:
`SELF` is now `private`.